src/test/commentCounter.test.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * @jest-environment jsdom
+ */
+
+describe('commentCounter', () => {
+  test('returns the number of comments for each movie', () => {
+    const data = [
+      {
+        comment: 'This is nice!',
+        creation_date: '2021-01-10',
+        username: 'John',
+      },
+      {
+        comment: 'Great content!',
+        creation_date: '2021-02-10',
+        username: 'Jane',
+      },
+    ];
+    const span = document.createElement('span');
+    span.innerHTML = data;
+    const result = (data).length;
+    expect(result).toBe(2);
+  });
+});
```

Comment:
You did a great job implementing the different project requirement 
- [ ] Please your counter functions  fail all the unit test. Kindly correct all the test 
![checkOut](https://user-images.githubusercontent.com/55596970/219871117-d92e6230-e383-49a8-9989-ba4e64497a41.png)

Reply:
Thank you for your review, Collins.


 **PASS  src/test/commentCounter.test.js
 PASS  src/test/moviecount.test.js

Test Suites: 2 passed, 2 total
Tests:       2 passed, 2 total
Snapshots:   0 total
Time:        0.604 s, estimated 1 s
Ran all test suites.
michaels-mbp:RS-Movie-App michaelobinyan$ npm test

> test
> jest

 PASS  src/test/commentCounter.test.js
 PASS  src/test/moviecount.test.js

Test Suites: 2 passed, 2 total
Tests:       2 passed, 2 total
Snapshots:   0 total
Time:        0.534 s, estimated 1 s
Ran all test suites.**

This is the result I am getting on my terminal. 
Can you please let me know what you what me to do differently?

Thank you.

Reply:
Hello @RitaDee 
Please can we have a zoom call

Reply:
Yes please.
Thank you

Reply:
Ok